docs/launch.md

```diff
@@ -33,6 +33,14 @@ If you like to interact with your collator through the [Polkadot JS UI](https://
 
 See [the collator section](./collator.md) to learn more about collator node setup and launch.
 
+## Local Development
+
+If you wish to simply run EduChain locally, you can use pop! (install it here).  Once installed, you can utilize the included Zombienet config.  This will spawn a relay chain, parachain collators, which are all accessible through PolkadotJS:
```

Comment:
```suggestion
To run and test Educhain locally, `pop-cli` can be used (install it [here](https://github.com/r0gue-io/pop-cli)).  Once installed, you can customize the Zombienet configuration included. This will spawn a relay chain node and parachain collator nodes, which can be accessed through Polkadot JS UI:
```